se.py

```diff
@@ -1,72 +0,0 @@
-from keras.layers import GlobalAveragePooling2D, Reshape, Dense, multiply, add, Permute, Conv2D
-from keras import backend as K
-
-
-def squeeze_excite_block(input, ratio=16):
```

Comment:
This file should not be removed. While the scripts no longer depend on it, it needs to be a standalone module as it was before for easy usage I'm custom models.